### PR TITLE
new-order does not throw error

### PR DESCRIPTION
**Pull request by Spiros-N-Agathos**
Modified new order query to not throw exception so that we do not pollute the psql log.

### CODE REVIEW

File: benchmarks/htap/schemas/common/new_order.sql [hunk 1]
```diff
@@ -9,28 +9,8 @@ CREATE FUNCTION new_order(
   , in_supware INT[]
   , in_qty INT[]
   , in_timestamp TIMESTAMPTZ
-) RETURNS TABLE(
```
Comment (lucvlaming) [inline benchmarks/htap/schemas/common/new_order.sql:12]: can we do this also with the other functions?

File: benchmarks/htap/lib/transactions.py [hunk 2]
```diff
@@ -66,6 +66,17 @@ def execute_sql(self, sql, args, query_type):
             else:
                 raise
 
+    def execute_sql_new_order(self, sql, args):
+        if self.dry_run:
+            return
+        start = time.time()
+        self.conn.cursor.execute(sql, args)
```
Comment (lucvlaming) [inline benchmarks/htap/lib/transactions.py:73]: this should be run in a transaction.

File: benchmarks/htap/lib/transactions.py [hunk 3]
```diff
@@ -66,6 +66,17 @@ def execute_sql(self, sql, args, query_type):
...
+        result = self.conn.cursor.fetchone()
+        if result == True:
+            self.add_stats('new_order', 'ok', start)
+        else:
+            self.add_stats('new_order', 'error', start)
```
Comment (lucvlaming) [inline benchmarks/htap/lib/transactions.py:78]: here you should abort the transaction

File: benchmarks/htap/lib/transactions.py (hunk 2 above)
Comment (Spiros-N-Agathos) [reply benchmarks/htap/lib/transactions.py:77]: Done

File: benchmarks/htap/lib/transactions.py (hunk 3 above)
Comment (Spiros-N-Agathos) [reply benchmarks/htap/lib/transactions.py:85]: Done

File: benchmarks/htap/schemas/common/new_order.sql (hunk 1 above)
Comment (Spiros-N-Agathos) [reply benchmarks/htap/schemas/common/new_order.sql:12]: So `order_status` and `payment` can be `VOID` same as `delivery`, right?

Comment (lucvlaming) [reply benchmarks/htap/schemas/common/new_order.sql:12]: i think so yes?

Comment (Spiros-N-Agathos) [reply benchmarks/htap/schemas/common/new_order.sql:12]: Done

**Review by lucvlaming — APPROVED** [review]

**Pull request merged**